xcube_stac/store.py

```diff
@@ -110,22 +194,17 @@ def get_data_ids(
         include_attrs: Container[str] = None,
         **open_params
     ) -> Union[Iterator[str], Iterator[Tuple[str, Dict[str, Any]]]]:
-        """Get an iterator over the data resource identifiers for the
-        given type *data_type*. If *data_type* is omitted, all data
-        resource identifiers are returned. The data resource identifiers
-        follow the following structure:
+        """Get an iterator over the data resource identifiers. The data
+        resource identifiers follow the following structure:
 
             `collection_id_0/../collection_id_n/item_id/asset_id`
 
         Args:
-            data_type: If given, only data identifiers
-                that are available as this type are returned. If this is None,
-                all available data identifiers are returned. Defaults to None.
```

Comment:
Keep documenting `data_type` arg. Say that it isn't used (yet).

Reply:
...and that a warning is emitted, if it is used.

Reply:
I added the documentation.